core/src/main/resources/swagger/api-docs.yaml

```diff
@@ -4820,6 +4829,15 @@ components:
           type: boolean
           default: false
           description: Whether or not to create output columns if they would be empty and optional.  Defaults to false (columns are created).
+        monitoringScript:
+          type: string
+          description: Location of user-defined monitoring script that runs inside the task container
+        monitoringImage:
+          type: string
+          description: User-defined monitoring container that runs as a sibling to the task container
```

Comment:
Might want to state directly that we expect either this or `monitoringScript`, not both.

Reply:
We technically don't care what users do, they could monitor multiple ways at once. But definitely clarified 👍